src/scss/_mixins.scss

```diff
@@ -8,7 +8,7 @@
 
 	// https://github.com/philipwalton/flexbugs#flexbug-17 (for suport in ie 9/10)
 	@include oGridRespondTo(L) {
-		flex: 0 1 25%;
+		flex: 0 1 calc(25% - 12px);
```

Comment:
If this is a magic number, could we have a variable that explains it in its place?

Reply:
Ah, yep... it's based on the 6px padding each side. Will add a comment stating that.

Reply:
👍